public/thirdPartyLicenses.html

```diff
@@ -42,7 +42,7 @@
             padding: 0.7em;
             background: white;
             border-top: 1px solid silver;
-            xborder-right: 1px solid silver;
+            border-right: 1px solid silver;
```

Comment:
Please do not modify this file!

Reply:
Oh sorry, Actually it is showing error that's why I have fixed it.

Reply:
@GMishx I have reverted the changes.